mochimochi/classifier/binary/adam.hpp

```diff
@@ -72,19 +77,19 @@ public :
     return true;
   }
 
-  int predict(const Eigen::VectorXd& feature) const {
+  int predict(const Eigen::VectorXd& feature) const override {
```

Comment:
Why some algorithms don't have the adds of "override"?

Reply:
That's a mistake and I'm not sure why the compiler doesn't complain about it. All implementations of the virtual methods defined in the `BinaryOML` interface should have override!

Reply:
I have resolved this.